charts/istio/istio-ingress/templates/deployment.yaml

```diff
@@ -78,7 +78,7 @@ spec:
             memory: 128Mi
           limits:
             cpu: 2000m
-            memory: 1024Mi
+            memory: 2048Mi
```

Comment:
You comment doesn't show such high resource usage:
```
kubectl -n istio-ingress top pods
NAME                                    CPU(cores)   MEMORY(bytes)
istio-ingressgateway-5889b5b484-5hfbd   281m         534Mi
istio-ingressgateway-5889b5b484-657wx   279m         173Mi
istio-ingressgateway-5889b5b484-d2qk9   338m         273Mi
istio-ingressgateway-5889b5b484-p7pr2   408m         638Mi
istio-ingressgateway-5889b5b484-rhzt5   388m         622Mi
```
Can you add proof for OOMKills, like e.g. events?

Reply:
```
Name:         istio-ingressgateway-5889b5b484-p7pr2
Namespace:    istio-ingress
Priority:     0
Start Time:   Fri, 26 Feb 2021 22:07:19 +0200
Labels:       app=istio-ingressgateway
              istio=ingressgateway
              pod-template-hash=5889b5b484
              service.istio.io/canonical-name=istio-ingressgateway
              service.istio.io/canonical-revision=1.7
Status:       Running
IP:           10.243.144.25
IPs:
  IP:           10.243.144.25
Controlled By:  ReplicaSet/istio-ingressgateway-5889b5b484
Containers:
  istio-proxy:
    Ports:         15021/TCP, 8443/TCP, 9443/TCP, 8132/TCP
    Host Ports:    0/TCP, 0/TCP, 0/TCP, 0/TCP
    Args:
      proxy
      router
      --domain=istio-ingress.svc.cluster.local
      --proxyLogLevel=warning
      --proxyComponentLogLevel=misc:error
      --log_output_level=all:warn,ads:error
      --drainDuration=45s
      --parentShutdownDuration=1m0s
      --connectTimeout=10s
      --serviceCluster=istio-ingressgateway
      --proxyAdminPort=15000
      --statusPort=15021
      --controlPlaneAuthPolicy=NONE
      --discoveryAddress=istiod.istio-system.svc:15012
    State:          Running
      Started:      Sat, 13 Mar 2021 03:56:14 +0200
    Last State:     Terminated
      Reason:       OOMKilled
      Exit Code:    0
      Started:      Mon, 08 Mar 2021 09:57:55 +0200
      Finished:     Sat, 13 Mar 2021 03:56:12 +0200
    Ready:          True
    Restart Count:  4
    Limits:
      cpu:     2
      memory:  1Gi
    Requests:
      cpu:      100m
      memory:   128Mi
```